types/prettier/index.d.ts

```diff
@@ -22,16 +22,16 @@ export type LiteralUnion<T extends U, U = string> = T | (Pick<U, never> & { _?:
 export type AST = any;
 export type Doc = doc.builders.Doc;
 
-// https://github.com/prettier/prettier/blob/master/src/common/fast-path.js
-export interface FastPath<T = any> {
+// https://github.com/prettier/prettier/blob/main/src/common/ast-path.js
+export interface AstPath<T = any> {
```

Comment:
Please add an alias for backward compatibility.

Reply:
I already thought about that, but it's a change from `2.2` to `2.3`
So if someone updates to the new types, they should just exchange their used `FastPath`
It's also a really quick search and replace

Otherwise I would just add `@deprecated` to `FastPath` and it would be marked for later removal, so why not just now directly

Reply:
Adding an alias is just one line and you indeed can mark it with `@deprecated`. Please do that.